whalesong/managers/status_v3.py

```diff
@@ -0,0 +1,109 @@
+from typing import Any, Dict, List, Optional
+from dirty_models import ModelField, HashMapField, ArrayField, IntegerField, StringIdField
+
+from . import BaseModelManager, BaseCollectionManager
+from .contact import Contact, ContactManager
+from ..models import BaseModel, DateTimeField
+from ..driver import BaseWhalesongDriver
+from ..results import Result, IteratorResult
+
+
+class StatusV3(BaseModel):
+    """
+    StatusV3 model
+    """
+
+    unread_count = IntegerField()
+    """
+    Unread statuses
+    """
+
+    expire_ts = DateTimeField()
+    """
+    Status expiration date
+    """
+
+    contact = ModelField(model_class=Contact)
+    """
+    Contact object
+    """
+
+    last_received_key = StringIdField()
+    """
+    Last encryption key received (¿?).
+    """
+
+    read_keys = HashMapField(field_type=StringIdField())
+
+
+class StatusV3Manager(BaseModelManager[StatusV3]):
+    """
+    StatusV3 manager. Allow to manage a WhatsApp status.
+
+    .. attribute:: msgs
+        :class: `~whalesong.managers.message.MessageCollectionManager`
+        StatusV3 message collection manager.
+
+    .. attribute:: contact
+        :class: `~whalesong.managers.contact.ContactManager`
+        StatusV3 contact manager.
+    """
+    MODEL_CLASS = StatusV3
+
+    def __init__(self, driver: BaseWhalesongDriver, manager_path: str = ''):
+        super(StatusV3Manager, self).__init__(driver=driver, manager_path=manager_path)
+
+        from .message import MessageCollectionManager
+        self.add_submanager('msgs', MessageCollectionManager(driver=self._driver,
+                                                             manager_path=self._build_command('msgs')))
+
+        self.add_submanager('contact', ContactManager(driver=self._driver,
+                                                      manager_path=self._build_command('contact')))
+
+    def send_read_status(self, message_id) -> Result[bool]:
+        """
+        Mark a statusV3 as read.
+
+        :param message_id: Message serialized ID to be marked
+        """
+        params = {
+            'messageId': message_id,
+        }
+
+        return self._execute_command('sendReadStatus', params)
+
+
+class StatusV3CollectionManager(BaseCollectionManager):
+    """
+    Manage a collection of StatusV3.
+    """
+    MODEL_MANAGER_CLASS = StatusV3Manager
+
+    def get_unexpired(self, is_unread) -> IteratorResult[StatusV3]:
```

Comment:
`is_unread` should be typed. Why do you use `is_unread` and not `unread`? What do you think about to set a default value? maybe `True`...